.github/blunderbuss.yml

```diff
@@ -3,7 +3,7 @@ assign_issues_by:
   - 'api: bigtable'
   - 'api: datastore'
   - 'api: firestore'
-  to: 
```

Comment:
This is an auto-change, as there was an unwanted whitespace, which got included as part of this change.